canonical-websites/partners.ubuntu.com
# Create partners.u.c-authors group in migration

Create the launchpad openid auth group in the initial migration.
Its existence is needed to request membership status from launchpad.
# QA
1. Run `make run` in a clean environment
2. Go to http://127.0.0.1:8003/admin
3. Ensure it asks for your partners.u.c-authors membership, and successfully logs in.
